setup.py

```diff
@@ -53,7 +53,8 @@ def read_files(*filenames):
         'django>=1.8;python_version>="3"',
         'six',
         'pillow<4;python_version<"2.7"',
-        'pillow;python_version>="2.7"',
+        'pillow<7;python_version<="2.7"',
+        'pillow;python_version>="3"',
```

Comment:
I think you can drop the `'pillow<4;python_version<"2.7"'` bit as easy-thumbnails doesn't claim to support Python < 2.7.

```
'pillow<7;python_version<"3"',
'pillow;python_version>="3"',